content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -1018,12 +1020,12 @@ Used to restart an archived chat.
 
 </Section>
 
-<!-- > **`activate_chat`** sample **request** with optional params
+<!-- > **`resume_chat`** sample **request** with optional params
```

Comment:
This is some legacy stuff that doesn't need to be updated. Feel free to delete it next time you come across it.